### PR TITLE
misc: fix broken links in changelog

### PR DESCRIPTION
**Pull request by takenspc**
**Summary**

"Full Changelog" links of changelogs are broken since v9.3.0.

* changelog.md
	* [9.5.0](https://github.com/GoogleChrome/lighthouse/blob/master/changelog.md#950-2022-03-09)
	* [9.4.0](https://github.com/GoogleChrome/lighthouse/blob/master/changelog.md#940-2022-02-16)
	* [9.3.1](https://github.com/GoogleChrome/lighthouse/blob/master/changelog.md#931-2022-01-31)
	* [9.3.0](https://github.com/GoogleChrome/lighthouse/blob/master/changelog.md#930-2022-01-31)
* GitHub Release pages
	* https://github.com/GoogleChrome/lighthouse/releases/tag/v9.6.2
	* https://github.com/GoogleChrome/lighthouse/releases/tag/v9.6.1
	* https://github.com/GoogleChrome/lighthouse/releases/tag/v9.6.0
	* https://github.com/GoogleChrome/lighthouse/releases/tag/v9.5.0
	* https://github.com/GoogleChrome/lighthouse/releases/tag/v9.4.0
	* https://github.com/GoogleChrome/lighthouse/releases/tag/v9.3.1
	* https://github.com/GoogleChrome/lighthouse/releases/tag/v9.3.0

Expected urls are `https://github.com/GoogleChrome/lighthouse/compare/v1...v2` though actual urls are `https://github.com/compare/v1..v2`.

I guess that [updating `conventional-changelog-cli`](https://github.com/GoogleChrome/lighthouse/commit/a26c8ee6a6e0a1a012c74685e1bafe00f133e857) cause this issue.

This PR fixes changelog template to genearete correct urls and broken links of changelog.md.

**Related Issues/PRs**

This issue seems trivial and fix is straightforward so I skipped creating an issue. Let me know if I should create an issue.

### CODE REVIEW

**Review by adamraine — APPROVED** [review]
Good catch, thanks for the PR. I'll go ahead and fix all of the release pages you mentioned.

**Comment by adamraine** [comment]
@takenspc please sign the CLA

**Comment by rayjenn** [comment]
Thank you for the reply sir. Sent from Mail for Windows From: Adam RaineSent: Wednesday, June 15, 2022 1:52 PMTo: GoogleChrome/lighthouseCc: SubscribedSubject: Re: [GoogleChrome/lighthouse] misc(build): fix broken links in changelog (PR ***@***.*** approved this pull request.Good catch, thanks for the PR. I'll go ahead and fix all of the release pages you mentioned.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you are subscribed to this thread.Message ID: ***@***.***> 



	
		
			
				
			
		
		
			
				This email has been checked for viruses by Avast antivirus software.
				www.avast.com

File: build/changelog-generator/templates/header.hbs [hunk 1]
```diff
@@ -8,7 +8,10 @@
  <a name="{{version}}"></a>
 # {{version}} ({{date}})
 {{#if @root.linkCompare~}}
-  [Full Changelog]({{~@root.repoUrl}}/compare/{{previousTag}}...{{currentTag}})
+  [Full Changelog](
+    {{~#if @root.host}}{{~@root.host}}/{{/if~}}
+    {{~#if @root.owner ~}}{{@root.owner}}/{{/if~}}
```
Comment (connorjclark) [inline build/changelog-generator/templates/header.hbs:13]: do all these newlines work? `yarn changelog`

Comment (adamraine) [reply build/changelog-generator/templates/header.hbs:13]: Looked fine when I tried it:
```md
 <a name="9.5.0"></a>
# 9.5.0 (2022-06-15)
[Full Changelog](https://github.com/GoogleChrome/lighthouse/compare/v7.3.0...v9.5.0)

~~ TODO: https://chromiumdash.appspot.com/schedule ~~
We expect this release to ship in the DevTools of [Chrome XX](https://chromiumdash.appspot.com/schedule), and to PageSpeed Insights within 2 weeks.

## Notable Changes
~~ TODO: Move notable changes here ~~


## New Audits

* add work-during-interaction diagnostic ([#13982](https://github.com/GoogleChrome/lighthouse/pull/13982))
* add responsiveness metric for timespans ([#13917](https://github.com/GoogleChrome/lighthouse/pull/13917))
* lcp-lazy-loaded ([#12838](https://github.com/GoogleChrome/lighthouse/pull/12838))
* fr: uses-responsive-images-snapshot ([#12714](https://github.com/GoogleChrome/lighthouse/pull/12714))

```

**Pull request merged**